src/semantics.ts

```diff
@@ -253,6 +253,12 @@ export const semantics: Readonly<[H5PFieldGroup, H5PBehaviour, H5PL10n]> = [
         default: "Saved",
         type: H5PFieldType.Text,
       },
+      {
+        label: "Word limit exceeded label",
+        name: "dialogNoteLimitExceeded",
+        default: "Limit exceeded",
```

Comment:
Perhaps something less generic?

```suggestion
        default: "Too many words",
```

Reply:
Yes, I agree ✨